src/sentry/rules/actions/base.py

```diff
@@ -60,6 +63,39 @@ def after(self, event: Event, state: EventState) -> Generator[CallbackFuture, No
         pass
 
 
+class SentryAppEventAction(EventAction, abc.ABC):
+    """Abstract class to ensure that actions in SENTRY_APP_ACTIONS have all required methods"""
+
+    @abc.abstractmethod
+    def get_custom_actions(self, project: Project) -> Sequence[Mapping[str, Any]]:
+        raise NotImplementedError()
```

Comment:
I think the new preference for `abstractmethod`s is to just `pass` in the method body.